<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDialog</class>
 <widget class="QWidget" name="OptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="deprecationNotice">
     <property name="text">
      <string>&lt;p&gt;&lt;b&gt;NOTE:&lt;/b&gt; KeePassHTTP has been deprecated and will be removed in the future.&lt;br&gt;Please switch to keepassxc-browser instead! For help with migration, visit our &lt;a href=&quot;https://keepassxc.org/docs/keepassxc-browser-migration&quot;&gt;keepassxc-browser migration guide&lt;/a&gt;.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="enableHttpServer">
     <property name="toolTip">
      <string>This is required for accessing your databases from ChromeIPass or PassIFox</string>
     </property>
     <property name="text">
      <string>Enable KeePassHTTP server</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="showNotification">
         <property name="text">
          <string>Sh&amp;ow a notification when credentials are requested</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="bestMatchOnly">
         <property name="toolTip">
          <string>Only returns the best matches for a specific URL instead of all entries for the whole domain.</string>
         </property>
         <property name="text">
          <string>&amp;Return only best matching entries</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="unlockDatabase">
         <property name="text">
          <string>Re&amp;quest to unlock the database if it is locked</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="matchUrlScheme">
         <property name="toolTip">
          <string>Only entries with the same scheme (http://, https://, ftp://, ...) are returned.</string>
         </property>
         <property name="text">
          <string>&amp;Match URL schemes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sortByUsername">
         <property name="text">
          <string>Sort matching entries by &amp;username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sortByTitle">
         <property name="text">
          <string>Sort &amp;matching entries by title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeSharedEncryptionKeys">
         <property name="text">
          <string>R&amp;emove all shared encryption keys from active database</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeStoredPermissions">
         <property name="text">
          <string>Re&amp;move all stored permissions from entries in active database</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Password Generator</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="HttpPasswordGeneratorWidget" name="passwordGenerator" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="MessageWidget" name="warningWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="alwaysAllowAccess">
         <property name="text">
          <string>Always allow &amp;access to entries</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="alwaysAllowUpdate">
         <property name="text">
          <string>Always allow &amp;updating entries</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="searchInAllDatabases">
         <property name="toolTip">
          <string>Only the selected database has to be connected with a client.</string>
         </property>
         <property name="text">
          <string>Searc&amp;h in all opened databases for matching entries</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="supportKphFields">
         <property name="toolTip">
          <string>Automatically creating or updating string fields is not supported.</string>
         </property>
         <property name="text">
          <string>&amp;Return advanced string fields which start with &quot;KPH: &quot;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>HTTP Port:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="httpPort">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="inputMask">
            <string notr="true">d0000</string>
           </property>
           <property name="placeholderText">
            <string>Default port: 19455</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>KeePassXC will listen to this port on 127.0.0.1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HttpPasswordGeneratorWidget</class>
   <extends>QWidget</extends>
   <header>http/HttpPasswordGeneratorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MessageWidget</class>
   <extends>QWidget</extends>
   <header>gui/MessageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
